docs/admins/operations/incident-response.md

```
@@ -335,6 +363,37 @@ kubectl describe certificate <certificate-name> -n eduide-system
+**Check the order:**
+
+```bash
+kubectl -n eduide-system get certificate <name> \
+  -o jsonpath='{range .status.conditions[*]}{.type}={.status} {.message}{"\n"}{end}'
+```
+
+Two different situations look the same from outside.
+
+**Finalize failed.** The challenges validated and the CA then failed the final step:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)